_releases/3.12.0.md

```diff
@@ -0,0 +1,100 @@
+
+
+## New Features <img width="25" alt ="features" src="https://user-images.githubusercontent.com/28867440/42755172-d8046d70-88ff-11e8-917c-ed1fcf2aac8c.png">
+
+- As the Contract Admin, I want to invite users to contract and workspace in one click right on the Contract Page.
+Now, the Contract Admin is allowed to invite other users to a workspace and a contract simultaneously on the Contract Page.
+
+- Remove the `hideUserSupport` parameter in the Tenant's configuration.
+The `hideUserSupport` parameter is redundant. Therefore, it was expelled from the "Tenant-model". 
+Now, there is the `intercomAppId` parameter that should be used instead. 
+
+- Do not add the intercom script on the page, if the `tenant.hideUserSupport` parameter is set.
+Now, the `intercomAppId` parameter affects an overall appearance of the Intercom's System.
+
+- Invite to a Contract and a Workspace in one click.
+Now, the Contract Admin is allowed inviting any other user to a workspace and a contract simultaneously right on the Workspace Page.
+
+- Extend driver events to have a purpose message from a Pod's container status.
+Common issues related to running steps inside a flow are reflected in the `runlog`. For example, the `Out of Memory` issue will be addressed as *"Component run out of memory and terminated."* This approach can help with improving an overall debugging procedure.
+
+- Check roles usage in the `/v2/tenant/:id/roles`.
+An additional verifications were added for changing roles procedure in the Tenant: 
+    - It is restricted to remove a role from the "Tenant-policy" if any user/contract utilizes it.
+    - In case the role has to be removed from the "Tenant-policy", but one of the `contract.availableRoles` is still using it, the `409` error code will be returned. 
+    
+- Remove `InviteToken`.
+Code refactoring/improving. The `ContractInviteToken` was used as a primary mechanism for invites. However, it was a legacy compatibility code (`src/commons/lib/models/contract.js`) that loads the `InviteToken`. The following means that the `InviteToken` had to be deleted, to make it possible to remove all the code that uses it.
+
+- `Execstats` collection is too large.
+The session's duration ("time of life") for expired `execstats` collection was decreased to reduce Database flooding with unwanted records.
+
+## Bug Fixes <img width="25" alt="bugfix" src="https://user-images.githubusercontent.com/28867440/42694680-e35bc0c4-86bb-11e8-8d4d-d2b1708e9213.png">
+
+- Broken `POST v2/users` endpoint with specified `relationships.contracts`.
+This endpoint was changed: the section relation was removed. Now, the user is being created out of the contract's scope. The following means that he is not allowed to log into our platform. Therefore, this user has to be added to one of the contracts utilizing [Add a new member to the Contract's scope] (https://api.elastic.io/docs/v2/#add-a-new-member-to-the-contract's-scope)
```

Comment:
This one introduces a breaking change to the mentioned API endpoint. Would be nice it emphasize it with some label/bold text/etc..